pkg/operator/targetconfigcontroller/targetconfigcontroller_test.go

```diff
@@ -2,30 +2,186 @@ package targetconfigcontroller
 
 import (
 	"context"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"reflect"
 	"testing"
-
-	"k8s.io/apimachinery/pkg/util/sets"
+	"unsafe"
 
 	configv1 "github.com/openshift/api/config/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	configlistersv1 "github.com/openshift/client-go/config/listers/config/v1"
+	"github.com/openshift/cluster-kube-scheduler-operator/pkg/operator/v410_00_assets"
+	"github.com/openshift/library-go/pkg/operator/events"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/diff"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/kubernetes/scheme"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/tools/cache"
-
-	operatorv1 "github.com/openshift/api/operator/v1"
-	configlistersv1 "github.com/openshift/client-go/config/listers/config/v1"
-	"github.com/openshift/library-go/pkg/operator/events"
 )
 
 var codec = scheme.Codecs.LegacyCodec(scheme.Scheme.PrioritizedVersionsAllGroups()...)
 
+var configLow = &configv1.Scheduler{
```

Comment:
`configLowNodeUtilization` is longer but better to understand by its name, the same for other occurences of short names:
- `configHigh` -> `configHighNodeUtilization`
- `configNo` -> `configNoScoring`
- `kscLow` and alike
- `cmLow` and alike

Reply:
Addressed in commit 764f14d9c4a0402f6d17b2fa017d13257f6f2bff